configs/default/DIAT-MAMBAF722_I2C.config

```diff
@@ -120,7 +120,7 @@ feature OSD
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set mag_hardware = NONE
+set mag_hardware = AUTO
```

Comment:
`AUTO` is the default for all of these - they should just be omitted, which removes the risk of them getting stale when a parameter is renamed or dropped.

Reply:
Thanks.   Removed.